### PR TITLE
fix: check that payerChannel is not null when displaying contributors

### PR DESCRIPTION
**Pull request by nickreynolds**

### CODE REVIEW

File: packages/dapp/src/components/listing/ListingRedux.tsx [hunk 1]
```diff
@@ -253,7 +250,9 @@ class ListingPageComponent extends React.Component<
   };
 
   private openStoryDetails = (postId: string) => {
-    this.civilContext.fireAnalyticsEvent("listing story boost", "story details clicked", postId);
+    console.log("context: ", this.context);
+    console.log("civilcontext: ", this.civilContext);
```
Comment (sruddy) [inline packages/dapp/src/components/listing/ListingRedux.tsx]: Do you need these logs?

Comment (nickreynolds) [reply packages/dapp/src/components/listing/ListingRedux.tsx]: oh whoops I'll remove those

**Comment by dankins** [comment]
:tada: This PR is included in version 4.10.0 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**